portal-web/test/functional/com/liferay/portalweb/macros/OAuth2.macro

```diff
@@ -361,8 +361,8 @@ definition {
 				Navigator.gotoNavUnderline(navUnderline = "Resource scopes");
 			}
 
-			if (isSet(resourcePanels) && (${resourcePanels} != "")) {
```

Comment:
The if() is added by my previous commit, so removing this won't affect tests in other teams.